pkg/bigquery/config/seed/data.json

```diff
@@ -0,0 +1,59 @@
+{
+  "TASK_INSERT": {
+    "schema": {
```

Comment:
I suggest we move this to `resource` json-schema.

Reply:
I was thinking that the resource configuration should not have schema, as schema is more closely tied to the data being inserted. This will allow for a single connector to be used to insert value into different tables across different pipelines. Otherwise, each connector will be tied to one table and can not be re-used.

So, how about we move the tablename as well here. What do you think?

Reply:
Yeah, I think it is a trade off.

What if the table needs to be shared in different pipelines? If we put the schema settings in component config, which means the user need to setup the schema multiple times in different pipelines.

IMO, table schema is more close to a resource setting (resource configuration), not a data flow or param setting (component configuration)

Reply:
As decided, we have moved the schema to resource and removed the ability to create table.